modules/swagger-codegen/src/main/resources/rust/configuration.mustache

```diff
@@ -3,13 +3,15 @@ use hyper;
 
 pub struct Configuration<C: hyper::client::Connect> {
   pub base_path: String,
+  pub user_agent: Option<String>,
   pub client: hyper::client::Client<C>,
 }
 
 impl<C: hyper::client::Connect> Configuration<C> {
   pub fn new(client: hyper::client::Client<C>) -> Configuration<C> {
     Configuration {
       base_path: "{{{basePath}}}".to_owned(),
+      user_agent: {{#httpUserAgent}}Some("{{{.}}}".to_owned()){{/httpUserAgent}}{{^httpUserAgent}}None{{/httpUserAgent}},
```

Comment:
I will replace None with "Swagger-Codegen/#{VERSION}/rust" to make it conform to other generators.

Reply:
Updated the default user agent via 44853c3. Please take a look as I'm no expert in Rust.

cc @frol @farcaller @bjgill